draft-iab-protocol-maintenance.md

```diff
@@ -168,10 +168,10 @@ deployed protocol is immeasurably more useful than a perfect protocol
 specification. This is particularly true in early phases of system design, to
 which the robustness principle is best suited.
 
-As demonstrated by the IAB document "What Makes for a Successful Protocol?"
-{{?RFC5218}}, success or failure of a protocol depends far more on factors like
-usefulness than on technical excellence. Timely publication of protocol
-specifications, even with the potential for flaws, likely contributed
+As demonstrated by the IAB document "What Makes for a Successful
+Protocol?"&nbsp;{{?RFC5218}}, success or failure of a protocol depends far more
```

Comment:
Really?  I don't like doing this.  Non-breaking space ends up in the end product, which is terrible.

Reply:
I think they added this because otherwise you get two spaces in the TXT version because of the question mark. Maybe we need to file a bug against xml2rfc?